### PR TITLE
Implemented the EVP skeleton class. Issue #22

### PR DESCRIPTION
**Pull request by felix-zailskas**
Implemented a skeleton class for the EVP. This includes the following:
- The EstimatedValuePredictor class. This class now uses just LinearRegression to predict from dummy values. The function _estimate_value_ can be used to get the value of a lead given its ID.
- The database package is a placeholder for the database we will use in the end. For now, it includes a singleton instance of a DatabaseDummy object. This object uses the dummy_leads.json file as its data. The dummies in that file are in accordance to the data field definition. 
- The database package also contains the _models.py_ and _parsers.py_ files. Here _models.py_ contains models that we might retrieve or store in the database. The Lead class is especially important here. I used the [_pydantic_](https://docs.pydantic.dev/2.4/) library for automatic field validation.

### CODE REVIEW

**Comment by luccalb** [comment]
Looks good, maybe you could add a `tests/test_evp.py` file that checks the evp outputs and will run during testing stage.

**Comment by felix-zailskas** [comment]
> Looks good, maybe you could add a `tests/test_evp.py` file that checks the evp outputs and will run during testing stage.

Adjusted.

File: src/database/parsers.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            lead_value = LeadValue(
+                life_time_value=life_time_value,
+                customer_probability=customer_probability,
+            )
+        else:
+            lead_value = None
+
+        for income_value in AnnualIncome:
+            annual_income = income_value
+            if data["annual_income"] < income_value:
```
[the remark this answers is not in the document]
Comment (felix-zailskas) [reply src/database/parsers.py:31]: If the lead has an income > 5.000.000$ then annual_income will become AnnualIncome.Class10. However, if for some reason the Enum has no values this variable will be undefined. Hence, I will adjust the code. I will also add tests for the parser.

[the remark this answers is not in the document]
Comment (luccalb) [reply src/database/parsers.py:31]: My mistake, `annual_income` is assigned before checking the threshold..

**Review by luccalb — APPROVED** [review]
Looks good!

**Pull request merged**